setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='django-voting',
-    version='0.2.1',
+    version='0.3.1',
```

Comment:
should this be `0.3.0`?

Reply:
It turns out that the branch in `omni/django-1.11` was released to the pypi as `0.3.0`. I think it's best if this was sanitized by bumping up to `v0.3.1` in `master` and continued from there. All the changes in that `v0.3.0` are also a part of this PR, so it should be fine.